jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -739,6 +744,10 @@ void ScienceArm::arm_preset_callback(std::string channel, ArmPreset msg) {
     go_to_target_angles(new_msg);
 }
 
+void ScienceArm::custom_preset_callback(std::string channel, CustomPreset msg) {
+    arm_state.set_preset_position(msg.preset);
+}
+
```

Comment:
Is there a reason this function can't be a member of the parent class instead of the children?